jsettlers.graphics/src/main/java/jsettlers/graphics/map/draw/MapObjectDrawer.java

```diff
@@ -338,10 +342,6 @@ private void drawObject(int x, int y, IMapObject object, float color) {
 			drawByProgress(x, y, 13, 42, progress, color);
 			break;
 
-		case WINE:
```

Comment:
It was used a long time ago. But we then introduced WINE_GROWING, WINE_HARVESTABLE, WINE_DEAD (like it is also done for CROP_*)